frontend/src/components/CombatResults.vue

```diff
@@ -184,9 +199,116 @@ export default Vue.extend({
 .crypto-warrior-image {
   max-width: 13em;
 }
+
+
+.bb{
+  border-bottom: 1px solid rgba(255, 255, 255, 0.282);
+  width: 70%;
+  margin: auto;
+}
+
+.bt{
+  border-top: 1px solid rgba(255, 255, 255, 0.282);
+
```

Comment:
bb?
bt? 

Can they be slightly more descriptive or added to a css class that requires them specifically